src/style.css

```diff
@@ -0,0 +1,138 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.page__main {
+  width: 198px;
+  height: 406px;
+}
+
+.main {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main__picture--margin_top {
+  margin-top: 32px;
+}
+
+.picture {
+  display: flex;
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+  padding-bottom: 19px;
+}
+
+.main__content {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+}
+
+.content__explanation {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.content__code {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.content__code--margin_top {
+  margin-top: 4px;
+}
+
+.content__reviews {
+  font-size: 10px;
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.content__reviews--margin_top {
+  margin-top: 16px;
+}
+
+.reviews__stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star-active.svg);
+  background-size: cover;
+}
+
+.stars__star--margin {
+  margin: 2px;
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
```

Comment:
- Reuse stars block from Stars task and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.